records_mover/db/postgres/copy_options/date_input_style.py

```diff
@@ -1,42 +1,22 @@
 from records_mover.utils import quiet_remove
 from records_mover.records.hints import cant_handle_hint
 from records_mover.records.types import RecordsHints
-from typing import Optional, Set, Union, TYPE_CHECKING
-if TYPE_CHECKING:
-    from typing_extensions import Literal
-    # https://www.postgresql.org/docs/9.5/runtime-config-client.html#GUC-DATESTYLE
-    #
-    # DateStyle (string)
-    #
-    #  Sets the display format for date and time values, as well as
-    #  the rules for interpreting ambiguous date input values. For
-    #  historical reasons, this variable contains two independent
-    #  components: the output format specification (ISO, Postgres,
-    #  SQL, or German) and the input/output specification for
-    #  year/month/day ordering (DMY, MDY, or YMD). These can be set
-    #  separately or together. The keywords Euro and European are
-    #  synonyms for DMY; the keywords US, NonEuro, and NonEuropean are
-    #  synonyms for MDY. See Section 8.5 for more information. The
-    #  built-in default is ISO, MDY, but initdb will initialize the
-    #  configuration file with a setting that corresponds to the
-    #  behavior of the chosen lc_time locale.
-    DateInputStyle = Union[Literal["DMY"], Literal["MDY"]]
-else:
-    DateInputStyle = str
-
-
-def determine_date_input_style(unhandled_hints: Set[str],
```

Comment:
I got the terminology slightly wrong - this is a date order style in postgres terminology.  It's only really relevant on import.